src/libs/actions/IOU.ts

```diff
@@ -3163,6 +3279,15 @@ function getSendMoneyParams(
                 },
             },
         },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${optimisticTransactionThread.reportID}`,
+            value: {
+                [optimisticCreatedActionForTransactionThread.reportActionID]: {
+                    pendingAction: null,
+                },
+            },
+        },
     ];
 
     const failureData: OnyxUpdate[] = [
```

Comment:
Do we need to update failureData with data for the transaction thread and the created report action like we're doing in buildOnyxDataForMoneyRequest?

Reply:
Yes, I'll add those in.